Pig Stack Trace
---------------
ERROR 6000: 
<file run_pig.hql, line 5, column 0> Output Location Validation Failed for: '/user/trace/pig/payinfo_t-diff_data More info to follow:
Output directory hdfs://hdpmaster1.tm.com.my:8020/user/trace/pig/payinfo_t-diff_data already exists

org.apache.pig.impl.plan.VisitorException: ERROR 6000: 
<file run_pig.hql, line 5, column 0> Output Location Validation Failed for: '/user/trace/pig/payinfo_t-diff_data More info to follow:
Output directory hdfs://hdpmaster1.tm.com.my:8020/user/trace/pig/payinfo_t-diff_data already exists
	at org.apache.pig.newplan.logical.visitor.InputOutputFileValidatorVisitor.visit(InputOutputFileValidatorVisitor.java:95)
	at org.apache.pig.newplan.logical.relational.LOStore.accept(LOStore.java:66)
	at org.apache.pig.newplan.DepthFirstWalker.depthFirst(DepthFirstWalker.java:64)
	at org.apache.pig.newplan.DepthFirstWalker.depthFirst(DepthFirstWalker.java:66)
	at org.apache.pig.newplan.DepthFirstWalker.depthFirst(DepthFirstWalker.java:66)
	at org.apache.pig.newplan.DepthFirstWalker.depthFirst(DepthFirstWalker.java:66)
	at org.apache.pig.newplan.DepthFirstWalker.walk(DepthFirstWalker.java:53)
	at org.apache.pig.newplan.PlanVisitor.visit(PlanVisitor.java:52)
	at org.apache.pig.newplan.logical.relational.LogicalPlan.validate(LogicalPlan.java:212)
	at org.apache.pig.PigServer$Graph.compile(PigServer.java:1851)
	at org.apache.pig.PigServer$Graph.access$300(PigServer.java:1527)
	at org.apache.pig.PigServer.execute(PigServer.java:1440)
	at org.apache.pig.PigServer.executeBatch(PigServer.java:488)
	at org.apache.pig.PigServer.executeBatch(PigServer.java:471)
	at org.apache.pig.tools.grunt.GruntParser.executeBatch(GruntParser.java:172)
	at org.apache.pig.tools.grunt.GruntParser.parseStopOnError(GruntParser.java:235)
	at org.apache.pig.tools.grunt.GruntParser.parseStopOnError(GruntParser.java:206)
	at org.apache.pig.tools.grunt.Grunt.exec(Grunt.java:81)
	at org.apache.pig.Main.run(Main.java:503)
	at org.apache.pig.Main.main(Main.java:178)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.hadoop.util.RunJar.run(RunJar.java:233)
	at org.apache.hadoop.util.RunJar.main(RunJar.java:148)
Caused by: org.apache.hadoop.mapred.FileAlreadyExistsException: Output directory hdfs://hdpmaster1.tm.com.my:8020/user/trace/pig/payinfo_t-diff_data already exists
	at org.apache.hadoop.mapreduce.lib.output.FileOutputFormat.checkOutputSpecs(FileOutputFormat.java:146)
	at org.apache.pig.newplan.logical.visitor.InputOutputFileValidatorVisitor.visit(InputOutputFileValidatorVisitor.java:69)
	... 25 more
================================================================================
